LoRaEngine/LoraKeysManagerFacade/DeviceFeedConstants.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    /// <summary>
+    /// These constants are use when user selected the DEPLOY_DEVICE option while created a Edge Gateway
+    /// This is for demo purpose only and should not be used for productive workloads.
+    /// </summary>
+    internal class DeviceFeedConstants
+    {
+        public const string AbpDeviceId = "46AAC86800430028";
```

Comment:
I would like to see primitives like `DevEui` being used in new code rather than degenerating to strings.

Reply:
They're anyway transformed as string later, so what will be the gain to have them in something else that string?